src/mapping/cellranger_multi/script.py

```diff
@@ -147,32 +170,29 @@ def generate_dict_category(name: str, args: dict[str, str]) -> list[str]:
     else:
         return []
 
-def generate_csv_category(name: str, args: dict[str, str]) -> list[str]:
-    title = [ f'[{name}]' ]
-    if len(args) > 0:
-        values = [ pd.DataFrame(args).to_csv(index=False) ]
-        return title + values + [""]
-    else:
+def generate_csv_category(name: str, args: dict[str, str], orient: str) -> list[str]:
+    assert orient in ("index", "columns")
+    if not args:
         return []
+    title = [ f'[{name}]' ]
+    # Which index to include in csv section is based on orientation
+    to_csv_args = {"index": (orient=="index"), "header": (orient=="columns")}
+    values = [pd.DataFrame.from_dict(args, orient=orient).to_csv(**to_csv_args).strip()]
+    return title + values + [""]
+
 
 def generate_config(par: dict[str, Any], fastq_dir: str) -> str:
-    serialized_refs = []
-    for reference in REFERENCES:
-        pars = subset_dict(par, {reference: 'ref'})
-        serialized = generate_dict_category(REFERENCE_CONFIG_KEYS[reference], pars)
```

Comment:
`generate_dict_category()` isn't (/shouldn't) be used anymore at this stage, but I don't see it having been removed